src/lib/state.ts

```diff
@@ -120,6 +121,7 @@ interface MetadataState {
   description?: string;
   label?: string;
   ccSpecific?: Record<string, any>;
+  valueChangeOptions?: (keyof ValueChangeOptions)[];
```

Comment:
`keyof` seems weird here.

Doing `keyof { a: number, b: string }` will return type `"a" | "b"` (note, no values). Is that what you expect or do you want to use `Partial` maybe?

Reply:
Looked at linked PR, this is ok.